Update edge-transport-servers.md (#2937)
MicrosoftDocs/OfficeDocs-Exchange: Exchange/ExchangeHybrid/edge-transport-servers.md

If you add an Edge Transport server to your hybrid deployment, you don't need to route mail sent between on-premises users and internet recipients through it. Only messages sent between the on-premises and Exchange Online organizations will be routed through the Edge Transport server.

> After running the HCW, ensure to update the Receive Connector on the Edge to ensure it will accept mail from EOP securely:

Reviewer: ```suggestion
After running the Hybrid Configuration wizard, update the Receive connector on the Edge Transport server to ensure it will accept mail from EOP securely:
```

> For Commercical Office 365 Tenant use:

Reviewer: ```suggestion
For Commercial Office 365 tenants, use:
```

> 

Reviewer: ```suggestion

```PowerShell
```

> Set-ReceiveConnector -Identity "<Edge server name>\Default internal receive connector <Edge server name>" -TlsDomainCapabilities mail.protection.outlook.com:AcceptOorgProtocol -Fqdn "subject name on the public cert on Edge"

Reviewer: ```suggestion
```PowerShell
Set-ReceiveConnector -Identity "<Edge server name>\Default internal receive connector <Edge server name>" -TlsDomainCapabilities mail.protection.outlook.com:AcceptOorgProtocol -Fqdn "subject name on the public cert on Edge"
```
```

> 

Reviewer: ```suggestion

```
```

> For US Government Office 365 Tenant use:

Reviewer: ```suggestion
For US Government Office 365 tenants, use:
```
